lib/util.js

```diff
@@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
 function isBuffer(arg) {
   return arg instanceof Buffer;
 }
-exports.isBuffer = isBuffer;
+exports.isBuffer = deprecate(isBuffer,
+    'util.isBuffer is deprecated, please use a user-land alternative.');
```

Comment:
You can also use Buffer.isBuffer which is not deprecated.
On Thu, Nov 12, 2015 at 3:55 AM hellboy81 <notifications@github.com> wrote:

> In lib/util.js
> <https://github.com/nodejs/node/pull/1301#discussion_r44648758>:
>
> > @@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
> >  function isBuffer(arg) {
> >    return arg instanceof Buffer;
> >  }
> > -exports.isBuffer = isBuffer;
> > +exports.isBuffer = deprecate(isBuffer,
> > +    'util.isBuffer is deprecated, please use a user-land alternative.');
>
> OK, should I use instanceof Buffer instead of util.isBuffer in Node 0.12.7
> and above?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/1301/files#r44648758>.
>